src/renderer/src/hooks/stores/mapeoDeviceStore.ts

```diff
@@ -20,16 +20,19 @@ export type MemberDevice = {
 type MapeoDeviceState = {
   nonMemberDevices: Record<Device['deviceId'], Device>
   memberDevices: Record<MemberDevice['deviceId'], MemberDevice>
+  projectName: string
```

Comment:
I added project name to zustand store as it is being used by multiple components